### PR TITLE
Create config.yml

### PR DESCRIPTION
**Pull request by emilyrohrbough**
Provide guidance for discussions, or Cypress App/Cloud issue location. 
pulled from: [cypress-io/cypress@develop/.github/ISSUE_TEMPLATE/config.yml](https://github.com/cypress-io/cypress/blob/develop/.github/ISSUE_TEMPLATE/config.yml?rgh-link-date=2023-03-14T19%3A20%3A17Z)
Screen Shot 2023-03-14 at 2 20 35 PM

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (emilyrohrbough) [inline .github/ISSUE_TEMPLATE/config.yml:1]: pulled from: https://github.com/cypress-io/cypress/blob/develop/.github/ISSUE_TEMPLATE/config.yml

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+blank_issues_enabled: false
+contact_links:
+  - name: 🤔 Questions and Help
+    url: https://github.com/cypress-io/cypress/discussions
+    about: This issue tracker is not for support questions. Please refer to our Discussions.
```
Comment (emilyrohrbough) [inline .github/ISSUE_TEMPLATE/config.yml:6]:
```suggestion
```

**Review by mjhenkes — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+blank_issues_enabled: false
+contact_links:
+  - name: 💬 Chat
+    url: https://on.cypress.io/discord
+    about: Want to discuss Cypress with others? Check out our chat.
+  - name: Cypress App or Cloud Issue
+    url: https://github.com/cypress-io/cypress-documentation/issues/new
```
Comment (jennifer-shehane) [inline .github/ISSUE_TEMPLATE/config.yml:8]: This is the docs repo link?

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (emilyrohrbough) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
    url: https://github.com/cypress-io/cypress/issues/new
```

**Review by jennifer-shehane — APPROVED** [review]

**Pull request merged**